sklearn/preprocessing/_encoders.py

```diff
@@ -590,8 +612,7 @@ def get_feature_names(self, input_features=None):
             if self.drop_idx_ is not None and self.drop_idx_[i] is not None:
                 names.pop(self.drop_idx_[i])
             feature_names.extend(names)
-
-        return np.array(feature_names, dtype=object)
```

Comment:
I am thinking that we might want to mention in the deprecation that we will not return a NumPy array but in list instead when one will change the method call.